### PR TITLE
Restock table and restock modals

### PR DESCRIPTION
**Pull request by lilianyangc**

### CODE REVIEW

File: src/components/Modal/ModalProducts.js [hunk 1]
```diff
@@ -100,11 +101,17 @@ function ModalProducts({
       onClose={() => setOpenModal(false)}
     >
       <DialogTitle style={{ paddingBottom: "0px" }}>
-        <Grid container>
+        <Grid container style={{ display: "flex" }} justify="space-between">
```
Comment (AshenCat) [inline src/components/Modal/ModalProducts.js:104]: grids are always display flex iirc

**Review by AshenCat — APPROVED** [review]

File: src/components/Modal/ModalProducts.js (hunk 1 above)
Comment (lilianyangc) [reply src/components/Modal/ModalProducts.js:104]: Oh really? It wasn't working without that.

Comment (AshenCat) [reply src/components/Modal/ModalProducts.js:104]: image
once you specify container within a grid, display flex will be set

Comment (lilianyangc) [reply src/components/Modal/ModalProducts.js]: Yeah, I followed that, that's why I added flex. LOL, Maybe it wasn't working at first coz the size 5 and 7, so it just made it fit. It just worked when I changed it to 5 and 5 and I didn't notice it XD

Comment (victafrancis) [reply src/components/Modal/ModalProducts.js]: 💯

**Review by victafrancis — APPROVED** [review]

**Pull request merged**